openbsd/docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
```

Comment:
@dteslya do you mind changing it to `debian:bookworm-slim`?
We are changing new images to this base image.

Reply:
Sure, will do.